src/main/java/dp/api/dataset/Alert.java

```diff
@@ -0,0 +1,48 @@
+package dp.api.dataset;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * An alert associated with a dataset version.
+ */
+public class Alert {
+
+    @JsonProperty("date")
+    private String date;
+
+    @JsonProperty("description")
+    private String description;
+
+    @JsonProperty("type")
+    private String type;
+
+    public Alert(String date, String description, String type) {
+        this.date = date;
+        this.description = description;
+        this.type = type;
+    }
+
+    public String getDate() {
+        return date;
+    }
+
+    public void setDate(String date) {
```

Comment:
Could this be time type instead of a string

Reply:
Need to check what the format is - will change if possible